app/dot/cpp/core/interfaces/BaseRequest.java

```diff
@@ -0,0 +1,3 @@
+package dot.cpp.core.interfaces;
+
+public interface BaseRequest {}
```

Comment:
this can extend Constraints.Validatable<String> as that has been the case for all requests until now